dev/main.tf

```diff
@@ -0,0 +1,13 @@
+module "dev" {
+  source = "github.com/hetznercloud/terraform-k8s-dev?ref=v0.3.0"
+
+  name         = "hccm-${replace(var.name, "/[^a-zA-Z0-9-_]/", "-")}"
+  worker_count = 1
+  # We deploy hccm through skaffold, its the application under development/test.
+  deploy_hccm      = false
+  use_cloud_routes = !var.setup_robot
```

Comment:
Should we use a consistent naming? 

- `deploy_hccm` => `hccm_enabled`
- `setup_robot` => `robot_enabled`